core/primitives/src/views.rs

```diff
@@ -320,6 +320,31 @@ pub struct ValidatorInfo {
     pub is_slashed: bool,
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct DebugChunkStatus {
+    pub shard_id: u64,
+    pub chunk_hash: ChunkHash,
+    pub gas_used: u64,
+    pub processing_time_ms: u64,
```

Comment:
Alternatively make it an Option. Or maybe skip serialisation if zero? Same in DebugBlockStatus.

Reply:
good idea - done